### ISSUE TITLE
Deploy to Cloud foundry error

### ISSUE THREAD

**Issue by ahmedmfarahat** [IBM/watson-google-assistant#36]
When I click on deploy on Cloud Foundry a toolchain gets created, however after a successful build and deployment stages, the app isn't running showing the following errors below:

**The app failed to initialize properly. Setup and restart needed. Error: Error. Unable to list workspaces for Assistant: Forbidden: Access is denied due to invalid credentials.**

message: 'Access is denied due to invalid credentials.',

body: '{"code":403,"error":"Forbidden"}',

**Comment by markstur** [comment]
I can reproduce. Trying to find out what happened.

**Comment by markstur** [comment]
@ahmedmfarahat Don't know how to fix the cloud yet, but here is a workaround...

Go to your IBM Cloud `Resource list` and find the Assistant service.  Not the CF one that is just a link but the plain ol' service that it links to.  (If you are unsure which one, then follow the app to the CF service and follow its link to the real service).  Click on the real assistant service and you should see a Manage tab showing an API key and a URL.  Copy those.

So the problem is that your CF app has a runtime environment variable (VCAP_SERVICES) with the wrong apikey.  We can fix the app by adding the correct runtime environment variables with the following keys:

key: ASSISTANT_URL  value: <the url you copied from the service>
key: ASSISTANT_APIKEY value <the api key you copied from the service>

It should look something like this (w/ my specific values here cut-off) and the when you save both values the app should restart and just work :)

![Screen Shot 2020-04-06 at 9 51 09 AM](https://user-images.githubusercontent.com/7129103/78584554-3813d600-77ed-11ea-9b25-e69f9f1b0a2e.png)

**Comment by ahmedmfarahat** [comment]
Yeah, but where can I change these keys found in the real service? please advice

**Comment by ahmedmfarahat** [comment]
I changed my WA credentials through cloud.ibm.com > resource list > your CF app > runtime environment, Added ASSISTANT_APIKEY AND ASSISTANT_URL and app restated automatically then worked perfectly. Thank you!
